src/users/users.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@nestjs/common';
+import { User } from './interfaces/user.interface';
+
+@Injectable()
```

Comment:
:wrench: Let's add a small hint that this service is not complete yet and will receive it's proper implementation in #26 similar to the comment above :up: 

```suggestion
//TODO: #26 - Overhaul Service methods according to use cases
@Injectable()
```